### DISCUSSION TITLE
在最后运行bot.py的时候出现这个问题

### DISCUSSION THREAD

**Discussion by User-Time** [FYWinds/takker#26] [Q&A] [answered]
[root@iZj6c1e5fhbe6ikkvlwsxjZ takker-v1.3.0]# poetry run /home/takker/takker-v1.3.0/bot.py 

  PermissionError

  [Errno 13] Permission denied: '/home/takker/takker-v1.3.0/bot.py'

  at /usr/local/lib/python3.9/os.py:597 in _execvpe
       593│         argrest = (args,)
       594│         env = environ
       595│ 
       596│     if path.dirname(file):
    →  597│         exec_func(file, *argrest)
       598│         return
       599│     saved_exc = None
       600│     path_list = get_exec_path(env)
       601│     if name != 'nt':
[root@iZj6c1e5fhbe6ikkvlwsxjZ takker-v1.3.0]# 

**/usr/local/lib/python3.9
/home/takker/takker-v1.3.0
/root/.poetry/**
这三个文件夹都给予了777权限

**Comment by FYWinds** [comment, marked as answer]
你漏了`python`，`poetry run`是在虚拟环境内执行 shell 指令而不是 python 指令
所以应该使用`poetry run python bot.py`
